sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -264,10 +285,14 @@ def load_static_information(self):
                         cursor.execute("SELECT CAST(ServerProperty('EngineEdition') AS INT) AS Edition")
                         result = cursor.fetchone()
                         if result:
-                            self.static_info_cache[STATIC_INFO_ENGINE_EDITION] = result
+                            self.static_info_cache[STATIC_INFO_ENGINE_EDITION] = result[0]
```

Comment:
```suggestion
                            self.static_info_cache[STATIC_INFO_ENGINE_EDITION] = result[0] if result else None
```

Just to be extra safe!

Reply:
We're already doing `if result:` before this though?

Reply:
disregard!